app/src/main/java/org/mozilla/focus/search/ManualAddSearchEnginePreference.kt

```diff
@@ -64,7 +65,7 @@ class ManualAddSearchEnginePreference(context: Context, attrs: AttributeSet) :
 
     override fun onRestoreInstanceState(state: Parcelable?) {
         val bundle = state as Bundle
-        super.onRestoreInstanceState(bundle.getParcelable(SUPER_STATE_KEY))
+        super.onRestoreInstanceState(bundle.getParcelableCompat(SUPER_STATE_KEY, Parcelable::class.java))
```

Comment:
I wonder why we started saving the super state ourselves.. 🤔

Reply:
Since 2018 😅 .
We could open a separate issue to investigate this.